packages/server/src/api/routes/projects.js

```diff
@@ -69,6 +69,9 @@ function createRouter(context) {
   router.get(
     '/:projectId/builds',
     handleAsyncError(async (req, res) => {
+      if (!isNaN(req.query.limit)) {
```

Comment:
nit: the coercion behavior of isNaN and parseInt are different and somewhat confusing, how about 

```suggestion
      if (Number.isInteger(parseInt(req.query.limit))) {
```

Reply:
Yes that's better, because negotiation is not so easy to read.